bin/github-pages

```diff
@@ -47,8 +47,10 @@ Mercenary.program(:"github-pages") do |p|
       puts "Checking domain #{cname}..."
       if check.valid?
         puts "Everything looks a-okay! :)"
+        exit 0
       else
         puts "Uh oh. Looks like something's fishy: #{check.reason}"
+        exit 1
```

Comment:
You might consider `abort "Uh oh."` as a means of consolidating these two lines :)